File: Java/1099.java
import java.util.Locale;
import java.util.Scanner;

public class Main {
    public static void main(String[] args) throws Exception {
        Locale.setDefault(Locale.US);
        Scanner input = new Scanner(System.in);

        int N, X, Y, soma = 0, MAX, MIN;

        N = input.nextInt();

        for (int i = 1; i <= N; i++) {

            X = input.nextInt();
            Y = input.nextInt();

            if (X > Y) {
                MAX = X;
                MIN = Y;
            }

            else {
                MAX = Y;
                MIN = X;
            }

            for (int j = MIN + 1 ; j < MAX; j++) {

                if (j % 2 != 0) {
                    soma += j;
                }

            } 

            System.out.println(soma);
            soma = 0;

        }

    }
}

File: Python/1099.py
N = int (input())

for i in range (0,N):
    X , Y = list(map(int,input().split()))
    soma = 0

    if X > Y:
        for j in range (Y+1,X):
            if j % 2:
                soma+=j

    else:
        for j in range (X+1,Y):
            if j % 2:
                soma+=j

    print(soma)
    